contracts/vaults/UniV3Vault.sol

```diff
@@ -225,27 +225,6 @@ contract UniV3Vault is IUniV3Vault, IntegrationVault {
         actualTokenAmounts[1] = amounts.a1;
     }
 
-    function _getLiquidityForAmounts(
```

Comment:
how's the best effort pull fixed if this is deleted?

Reply:
I think it is not a problem when UniV3Vault goes last.